src/main/java/io/anserini/collection/WashingtonPostCollection.java

```diff
@@ -103,45 +107,85 @@ private void parseRecord(String record) {
       bufferedRecord.articleUrl = wapoObj.getArticleUrl();
       bufferedRecord.author = wapoObj.getAuthor();
       bufferedRecord.obj = wapoObj;
-      bufferedRecord.content = record;
+      bufferedRecord.raw = record;
     }
   }
 
   /**
    * A document from the <a href="https://trec.nist.gov/data/wapost/">TREC Washington Post Corpus</a>.
    */
   public static class Document implements SourceDocument {
-    private static final Logger LOG = LogManager.getLogger(Document.class);
+    public static final List<String> CONTENT_TYPE_TAG = Arrays.asList("sanitized_html", "tweet");
 
     // Required fields
     protected String id;
     protected Optional<String> articleUrl;
     protected Optional<String> author;
     protected long publishDate;
     protected Optional<String> title;
-    protected String content;
+    protected String raw;
     protected WashingtonPostObject obj;
 
+    protected String fullCaption = null;
+    protected String kicker = null;
```

Comment:
What's a kicker?

Reply:
It's one of the fields in WashingtonPost corpus: http://trec-news.org/guidelines-2020.pdf

> ... we decree that articles from the "Opinion", "Letters to the Editor", or "The Post's View" sections, as labeled in the "kicker" field, are not relevant